### PR TITLE
Setup sdk

### PR DESCRIPTION
**Pull request by brosenberg42**
This change is [Reviewable](https://reviewable.io/reviews/openmpf/openmpf-python-component-sdk/1)

### CODE REVIEW

**Comment by jrobble** [comment]
The C++ Component SDK has this layout:

```
openmpf-cpp-component-sdk
+ detection
   + api
   + examples
```

The Java Component SDK has this layout:

```
openmpf-java-component-sdk
+ detection
  + java-component-api
  + examples
```

For consistency, this SDK should also have a "detection" directory.

---

Reviewed 5 of 11 files at r1.
Review status: 5 of 11 files reviewed at latest revision, all discussions resolved.

---

*[LICENSE, line 24 at r1](https://reviewable.io/reviews/openmpf/openmpf-python-component-sdk/1#-LCEjg6X85jwEeyHH58L:-LCEjg6X85jwEeyHH58M:b-xon2t8) ([raw file](https://github.com/openmpf/openmpf-python-component-sdk/blob/45b3c6ef7f9812abbe7b3289ce34d94d6ae16d54/LICENSE#L24)):*
Instead of:

> The above license applies to all of the Media Processing Framework (MPF) source code in the MPF core repository

Please reword this to:

> The above license applies to all of the Open Media Processing Framework (OpenMPF) source code in this repository

We need to make the same changes to the other SDK repos and openmpf repo. "MPF core repository" is too vague and doesn't apply to the SDK repos.

I created this: https://github.com/openmpf/openmpf/issues/559

---

*[README.md, line 46 at r1](https://reviewable.io/reviews/openmpf/openmpf-python-component-sdk/1#-LCEmPvU-AYmQeaTCtCW:-LCEmPvV1q8OQO6rOfxK:b5z3zvu) ([raw file](https://github.com/openmpf/openmpf-python-component-sdk/blob/45b3c6ef7f9812abbe7b3289ce34d94d6ae16d54/README.md#L46)):*
> ```Markdown
>     * [OpenMPF C++ Component sdk](https://github.com/openmpf/openmpf-cpp-component-sdk)
>     * [OpenMPF Java Component sdk](https://github.com/openmpf/openmpf-java-component-sdk)
>     * [OpenMPF Python Component sdk](https://github.com/openmpf/openmpf-python-component-sdk) ( **You are here** )
> ```

This "Where Am I?" section is great for end users, but every README.md needs to be updated when we add a new repo. I created this task: https://github.com/openmpf/openmpf/issues/561

---

*[README.md, line 58 at r1](https://reviewable.io/reviews/openmpf/openmpf-python-component-sdk/1#-LCEmgXS6ph1GK73EA-Q:-LCEmgXS6ph1GK73EA-R:btwe98e) ([raw file](https://github.com/openmpf/openmpf-python-component-sdk/blob/45b3c6ef7f9812abbe7b3289ce34d94d6ae16d54/README.md#L58)):*
> ```Markdown
> ### Using the Component SDK
> 
> TODO: Add link targets
> ```

I created this task for these TODOs: https://github.com/openmpf/openmpf/issues/560

---


*Comments from [Reviewable](https://reviewable.io/reviews/openmpf/openmpf-python-component-sdk/1#-:-LCEnCxN14tk2g5OZKGD:bbi0p8g)*

**Comment by jrobble** [comment]
Reviewed 6 of 11 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io/reviews/openmpf/openmpf-python-component-sdk/1)*

**Comment by brosenberg42** [comment]
Review status: 4 of 11 files reviewed at latest revision, 3 unresolved discussions.

---

*[LICENSE, line 24 at r1](https://reviewable.io/reviews/openmpf/openmpf-python-component-sdk/1#-LCEjg6X85jwEeyHH58L:-LCFLjrPBeSNtYZX9hiL:b-896fix) ([raw file](https://github.com/openmpf/openmpf-python-component-sdk/blob/45b3c6ef7f9812abbe7b3289ce34d94d6ae16d54/LICENSE#L24)):*
<details><summary><i>Previously, jrobble (Jeff Robble) wrote…</i></summary><blockquote>

Instead of:

> The above license applies to all of the Media Processing Framework (MPF) source code in the MPF core repository

Please reword this to:

> The above license applies to all of the Open Media Processing Framework (OpenMPF) source code in this repository

We need to make the same changes to the other SDK repos and openmpf repo. "MPF core repository" is too vague and doesn't apply to the SDK repos.

I created this: https://github.com/openmpf/openmpf/issues/559

</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io/reviews/openmpf/openmpf-python-component-sdk/1)*

**Comment by jrobble** [comment]
Reviewed 7 of 13 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io/reviews/openmpf/openmpf-python-component-sdk/1)*
